code/modules/mob/living/carbon/xenomorph/castes/Sentinel.dm

```diff
@@ -103,6 +103,6 @@
 #undef NEURO_TOUCH_DELAY
 
 /datum/behavior_delegate/sentinel_base/proc/paralyzing_slash(mob/living/carbon/human/human_target)
-	human_target.KnockDown(2)
-	human_target.Stun(2)
+	human_target.KnockDown(2.5)
+	human_target.Stun(2.5)
```

Comment:
i recomend using apply_effect , also Weaken should work better than knockdown+ stun